scripts/update-release-info.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# patch-releases.md
+####################################
+cat << EOF > content/en/releases/patch-releases.md
+---
+title: Patch Release Details
+type: docs
+auto_generated: true
+---
+<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/update-release-info.sh in k/website -->
+
+{{< warning >}}
+This content is auto-generated and links may not function. The source of the document is located [here](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md).
+{{< /warning >}}
+
+EOF
+
+curl --retry 3 https://raw.githubusercontent.com/kubernetes/sig-release/master/releases/patch-releases.md >> content/en/releases/patch-releases.md
```

Comment:
Worth noting that when branch names flip over to `main`, this script will break

Reply:
@celestehorgan -- We'll obviate the need for this ahead of any branch renames.